uberfire-widgets/uberfire-widgets-table/src/main/java/org/uberfire/ext/widgets/table/client/UberfireSimplePager.java

```diff
@@ -372,16 +373,16 @@ protected String createText() {
                              endIndex );
         boolean exact = display.isRowCountExact();
         if ( dataSize == 0 ) {
-            return "0 of 0";
+            return "0 " + of() + " 0";
         } else if ( pageStart == endIndex ) {
             return formatter.format( pageStart )
-                    + " of "
+                    + " " + of() + " "
                     + formatter.format( dataSize );
         }
         return formatter.format( pageStart )
                 + "-"
                 + formatter.format( endIndex )
-                + ( exact ? " of " : " of over " )
+                + ( exact ? " " + of() + " " : " " + of() + " over " )
                 + formatter.format( dataSize );
```

Comment:
@csadilek Yes, well spotted.. although the BZ only says "of" should be localized, I think we should show willing ;)

@paulovmr Sorry.. I missed that.